src/SFA.DAS.Reservations.Web/Stubs/ProviderRelationshipsApiClientStub.cs

```diff
@@ -27,29 +27,29 @@ public ProviderRelationshipsApiClientStub(IEncodingService encodingService)
                     new AccountProviderLegalEntityDto
                     {
                         AccountId = 1,
-                        AccountPublicHashedId = _encodingService.Encode(1,EncodingType.AccountId),
+                        AccountPublicHashedId = _encodingService.Encode(1,EncodingType.PublicAccountId),
                         AccountName = "Account 1",
```

Comment:
Do we need to change these account ids to 123 as well?

Reply:
I'd also set them in a const if they are reused in the code. Its a new change in how we deal with hard coded values